pyanaconda/network.py

```diff
@@ -64,6 +65,13 @@
 network_connected_condition = threading.Condition()
 
 
+class NetworkOnBoot(Enum):
+    """The default network on boot."""
```

Comment:
"""Device to be activated on boot if none was configured so."""
... don't like it much but can't come up with better.

Reply:
I would add `Network devices...`

Reply:
Fixed.